ui/app/AppLayouts/Chat/panels/ContactListPanel.qml

```diff
@@ -34,7 +35,14 @@ StatusListView {
 
         title: !model.displayName.endsWith(".eth") && !!model.localNickname ?
                     model.localNickname : Utils.removeStatusEns(model.displayName)
-        image.source: Global.getProfileImage(model.pubKey)
+        image.source: model.icon
+        icon: StatusIconSettings {
+            color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.pubKey)]
```

Comment:
It's better to check array borders here with some default color

Reply:
Can you show how you expect it to look like ? Also we are missing that kind of check on many other places.

Reply:
```
color: {
    const pubKeyColorId = Utils.colorIdForPubkey(model.pubKey);
    if (pubKeyColorId < 0 || pubKeyColorId >=  Theme.palette.userCustomizationColors) {
       // emit some warning
        return someDefaultColor;
   }
   return Theme.palette.userCustomizationColors[pubKeyColorId]
}
```
Maybe it's not necessary but if some one change Theme.palette this check helps to discover problem